src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+# Homework 2
+
+**Dataset description:**
+- orders.parquet: An order history of customers. Each row is an order and the item_ids for the order are stored as a list in the item_ids column
+- regulars.parquet: Users are allowed to specify items that they wish to buy regularly. This data gives the items each user has asked to get regularly, along with when they input that information.
+- abandoned_cart.parquet: If a user has added items to their basket but not
+bought them, we capture that information. Items that were abandoned are stored
+as a list in item_ids.
+- inventory.parquet: Some information about each item_id
+- users.parquet: Information about users.
+
+
+```python
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+```
+
+
+```python
+df_orders = pd.read_parquet('/Users/alvaroleal/Desktop/DS/Zrive/orders.parquet')
```

Comment:
Put the root folder in a variable`root_folder = /Users/alvaroleal/Desktop/DS/Zrive/` and then concatenate the name of each file, instead of repeting that root folder every time